File: dev/requirements/screen/cases/countdown-over.2.7.case.swift
#if canImport(SwiftUI) && os(iOS)
import SetlistUI

/// 2.7 — a minute past the slot: the countdown red, counting up.
let countdownOver_2_7 = RequirementCase(id: "2.7") {
    try MainActor.assumeIsolated {
        try expectScreen(StageScreen(state: StageFixture.state(elapsed: 1260)), slug: "countdown-over", id: "2.7", region: .clocks)
    }
}
#endif

File: dev/requirements/screen/cases/countdown-over.2.7.case.dart
import 'package:setlist_ui/setlist_ui.dart';

import '../../shared/requirement_case.dart';
import '../../shared/snapshot.dart';

/// 2.7 — Once the slot is over the countdown turns red and counts up.
final countdownOver_2_7 = RequirementCase('2.7', (tester) async {
  await expectRegion(
    tester,
    StageFixture.state(elapsed: const Duration(minutes: 21)),
    region: StageRegion.clocks,
    slug: 'countdown-over',
    id: '2.7',
  );
});
